src/EnergyPlus/DataSizing.cc

```diff
@@ -517,9 +517,11 @@ Real64 calcDesignSpecificationOutdoorAir(EnergyPlusData &state,
 
     auto &thisDSOA = state.dataSize->OARequirements(DSOAPtr);
 
-    if (thisDSOA.numDSOA == 1) {
+    if (thisDSOA.numDSOA == 0) {
```

Comment:
Fix DesignSpecification:OutdoorAir:SpaceList so it works if there is only one space in the list. Other changes in this file are for that.